src/scene/scene.js

```diff
@@ -349,8 +391,12 @@ Scene.prototype.applySettings = function (settings) {
     this._skyboxIntensity = settings.render.skyboxIntensity === undefined ? 1 : settings.render.skyboxIntensity;
     this._skyboxMip = settings.render.skyboxMip === undefined ? 0 : settings.render.skyboxMip;
 
-    this._resetSkyboxModel();
-    this.updateShaders = true;
+    if (settings.render.skyboxRotation !== undefined) {
+        this.skyboxRotation = new Quat(settings.render.skyboxRotation);
+    } else {
+        this._resetSkyboxModel();
+        this.updateShaders = true;
```

Comment:
should _resetSkyboxModel be in else branch?
when settings.render.skyboxRotation si undefined, it won't get called .. but it used to

Reply:
I'm using the object Object.defineProperty() accessors to do extra stuff in the set() accessor - which also calls the reset. So if settings.render.skyboxRotation is undefined - the _resetSkyboxModel is called - and if settings.render.skyboxRotation is defined the _resetSkyboxModel is called in the accessor - so behavior is correct - but maybe I can improve it by using a common private function...

Reply:
made improvements - thanks!